addons/website/static/src/js/menu/content.js

```diff
@@ -3,13 +3,216 @@ odoo.define('website.contentMenu', function (require) {
 
 var core = require('web.core');
 var Dialog = require('web.Dialog');
+var time = require('web.time');
 var weContext = require('web_editor.context');
 var widget = require('web_editor.widget');
 var websiteNavbarData = require('website.navbar');
+var websiteRootData = require('website.WebsiteRoot');
+var Widget = require('web.Widget');
 
 var _t = core._t;
 var qweb = core.qweb;
 
+var PagePropertiesDialog = widget.Dialog.extend({
```

Comment:
You must divide your class in lifecycle/public/private/handlers/... sections (see https://github.com/odoo/odoo/wiki/Javascript-coding-guidelines)

Reply:
It should be good now, I saw the 'save' method of Dialog is in --public-- and not with init/willStart/start?